app/uk/gov/hmrc/cardpaymentfrontend/services/CountriesService.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cardpaymentfrontend.services
+
+import uk.gov.hmrc.cardpaymentfrontend.models.Country
+import play.Environment
+import play.api.i18n.Messages
+import play.api.mvc.Request
+import uk.gov.hmrc.cardpaymentfrontend.requests.RequestSupport
+
+import javax.inject.{Inject, Singleton}
+import scala.io.Source
+
+@Singleton
+class CountriesService @Inject() (requestSupport: RequestSupport, env: Environment) {
+
```

Comment:
I wonder if we should instantiate a val here that is the list of countries.
Currently it calls the function getCountries every time the page is rendered. Since the list is hard coded in the bespoke messages file for these things I don't think we need to evaluate it every time - what are others thoughts?